react/components/EXPERIMENTAL_Table/hooks/useTableVisibility.ts

```diff
@@ -15,14 +15,11 @@ export default function useTableVisibility({
     return columns.reduce(reducer, [])
   }, [hiddenColumns, columns])
 
-  const toggleColumn = useCallback(
-    (id: string) => {
-      hiddenColumns.includes(id)
-        ? setHiddenColumns(hiddenColumns.filter(col => col !== id))
-        : setHiddenColumns([...hiddenColumns, id])
-    },
-    [hiddenColumns]
-  )
+  const toggleColumn = useCallback((id: string) => {
```

Comment:
Since the dependencies are empty, this `useCallback` is really necessary? I'm not sure about this, that's why I'm asking

Reply:
The `useCallback` makes possible to add the `toggleColumn` as an effect or callback dependency.